post.hbs

```diff
@@ -67,6 +67,19 @@ into the {body} of the default.hbs template --}}
                 </div>
                 {{/has}}
                 <hr />
+                <div class="post-full-author-header">
+
+                    {{!-- There are two options for how we display the byline/author-info.
+                If the post has more than one author, we load a specific template
+                from includes/byline-multiple.hbs, otherwise, we just use the
+                default byline. --}}
+
+                    {{#has author="freeCodeCamp.org"}}
+                    {{else}}
+                    {{> "byline-single"}}
+                    {{/has}}
+                </div>
+                <hr />
```

Comment:
We could conditionally render the `<hr>` element if the author is not freeCodeCamp.org.

```
{{^has author="freeCodeCamp.org"}}
<hr />
{{/has}}
```

Not sure if this is the best way, but I figure it's easier to style than having the `<hr>` in `byline-single.hbs`.